package.json

```diff
@@ -4,8 +4,10 @@
   "dependencies": {
```

Comment:
- [ ] Kindly check during the build it says these two modules `react-redux, axios` are not included in your `package.json`